WebNetSample/Views/Categories/Index.cshtml

```diff
@@ -29,7 +29,5 @@
             <th>Operations</th>
         </tr>
     </thead>
-    @await Component.InvokeAsync("CategoriesWidget");
-</table>
-
-@await RenderSectionAsync("Footer");
\ No newline at end of file
+    @await Component.InvokeAsync("CategoriesWidget")
```

Comment:
Probably you can use `nameof` section to set your widget name by original class name. Same case for other usages